internal/changelog/builder.go

```diff
@@ -161,6 +168,39 @@ func collectFragment(fs afero.Fs, path string, info os.FileInfo, err error, file
 	return nil
 }
 
+func ExtactEventNumber(linkType, eventURL string) (string, error) {
+	urlParts := strings.Split(eventURL, "/")
```

Comment:
Would be better to use URL parsing here? https://gobyexample.com/url-parsing https://pkg.go.dev/net/url#URL

Reply:
URL parsing unfortunately doesn't help much as I still have to split the link. I've added it as an additional validation for the link.

Reply:
I don't think it's necessary to add it as an additional step, I was wondering if it could be of help for instead of string matching, but as I see it is not.

Reply:
Alright, will also remove it.